src/Controller/Admin/ActivityController.php

```diff
@@ -80,6 +83,26 @@ public function indexAction(): Response
         ]);
     }
 
+    /**
+     * Lists all archived activities.
+     */
+    #[Route('/archive', name: 'archive', methods: ['GET'])]
+    public function indexArchiveAction(): Response
```

Comment:
Since this method mirrors the structure of `indexAction`, I'd suggest to use a query parameter instead, reducing code duplication.

Reply:
Partially see my respose [here](https://github.com/jasperweyne/helpless-kiwi/pull/325#discussion_r1157962931). While the method structure is (almost) identical now, it is imaginable that this could change.

Moreover, this way it's more convenient to render either of these pages.